Server/controllers/User/User.ts

```diff
@@ -18,13 +18,15 @@ export class UserController extends CrudController {
         throw new Error("Method not implemented.");
     }
 
-    public async read(req: Request<import("express-serve-static-core").ParamsDictionary>, res: Response): Promise<AlgoList>   {
+    public async read(req: Request<import("express-serve-static-core").ParamsDictionary>, res: Response, next:express.NextFunction):Promise<void>   {
```

Comment:
Changed this promise to Void. This should not return anything as it should next back to the userRouter

Reply:
Agreed.